macros/src/main/scala/packaged/insidePackageObject/package.scala

```diff
@@ -2,7 +2,7 @@ package packaged
 
 import scala.gestalt._
 
-package object macros2 {
+package object insidePackageObject {
   inline def plus(a: Int, b: Int): Int = meta {
     q"$a + $b"
   }
```

Comment:
@valdisxp1 is it a compiler bug that `plus` cannot defined inside the `macros` package? If that's the case, it would be good to report an error for the compiler.